packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -351,10 +352,21 @@ class ChipThemeData with Diagnosticable {
   /// Defaults to 4 logical pixels on all sides.
   final EdgeInsetsGeometry padding;
 
+  /// The color and weight of the chip's outline.
+  ///
+  /// If null, defaults to the value set by [shape].
+  ///
+  /// This value is combined with [shape] to create a shape decorated with an
+  /// outline.
+  final BorderSide? side;
+
   /// The border to draw around the chip.
   ///
   /// Defaults to a [StadiumBorder]. Must not be null.
```

Comment:
Can also be a MaterialProperty<OutlinedBorder> right?

Reply:
Yes, adding doc comment for it.